docs/pages/build/building-from-github.mdx

```diff
@@ -109,3 +116,101 @@ of the build in the PR's checks. A link to the build will be available in the ch
+/>
+
+You can also configure triggers for specific platforms and build profiles. If you select multiple
+platforms, a separate trigger will be made for each.
+
+<ImageSpotlight
+  alt="A filled-out version of the new build trigger form"
+  src="/static/images/eas-build/build-from-github/filled-build-trigger-form.png"
+/>
+
+<ImageSpotlight
+  alt="The build triggers section on the Expo project GitHub settings page filled with build triggers"
+  src="/static/images/eas-build/build-from-github/filled-build-triggers-table.png"
+/>
+
+When you push to a branch or tag, you can find the builds by looking at the “Checks” section on a
+commit.
+
+<ImageSpotlight
+  alt="The GitHub checks section on a branch commit"
+  src="/static/images/eas-build/build-from-github/builds-executed-automatically-on-branch.png"
+/>
+
+<ImageSpotlight
+  alt="The GitHub checks section on a tag commit"
+  src="/static/images/eas-build/build-from-github/tag-triggered-build.png"
+/>
+
+For pull requests, you can configure a “target branch pattern”. This is the destination branch of
+the pull request you want to build. The same rules apply for wildcards here as well.
+
+<ImageSpotlight
+  alt="The build trigger form for pull request"
+  src="/static/images/eas-build/build-from-github/pull-request-trigger-form.png"
+/>
+
+When you push to a pull request with a source and target branch matching this trigger, you'll find
+these builds in the checks section of the pull request:
+
+<ImageSpotlight
+  alt="The GitHub checks section on a pull request"
+  src="/static/images/eas-build/build-from-github/pull-request-triggered-build.png"
+/>
+
+> **Note:** To trigger builds from a pull request, the pull request's author must be a collaborator
+> on the GitHub repository. If you want to build pull requests from external contributors, [apply a PR Label](#build-using-github-pr-labels).
+
+#### Managing build triggers
+
+On your project's GitHub settings page on expo.dev, you can click the options button to the right of
+a build trigger row to manage that trigger. You can disable, edit, or delete the trigger from here.
+
+<ImageSpotlight
+  alt="Manage GitHub build trigger with the options button"
+  src="/static/images/eas-build/build-from-github/managing-build-triggers.png"
+/>
+
+### Troubleshooting
+
+- When things go wrong, we'll comment on the commit we attempted to build with some error information. We also show the latest result in the build triggers UI, which includes error information when you hover the “Error” tag.
+
+<ImageSpotlight
+  alt="Status UI on the build trigger UI"
+  src="/static/images/eas-build/build-from-github/trigger-troubleshooting-status.png"
+/>
+
+- Double-check that everything in the [Prerequisites](#prerequisites) section is true when trying to build.
+- Confirm that your base directory is accurate if you're using a monorepo setup.
+- Is your build profile correct? If a matching profile can't be found in eas.json, the build will not dispatch.
```

Comment:
```suggestion
- Double check everything in the [Prerequisites](#prerequisites) section is true when trying to build.
- Confirm that your base directory is accurate if you're using a monorepo setup.
- Is your build profile correct? If a matching profile can't be found in **eas.json**, the build will not dispatch.
```

[File names should be bold as per our writing style guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#filenames-as-bold-text)